tools/load-test.js

```diff
@@ -1,336 +0,0 @@
-/* eslint no-console: 0 */ // allow console.log() in this file
```

Comment:
I opted for a fresh start in our internal sysconf repo (mainly so I can easily get all our nice TS tooling). I'd be happy to move that over to the main repo at some point, but for now I'm going to remove this, as it won't work without changes.

Reply:
Yep, I'm happy to have this deleted.